hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtSoftwareModuleResourceTest.java

```diff
@@ -374,13 +373,57 @@ public void uploadArtifactsUntilQuotaExceeded() throws Exception {
 
     }
 
+    @Test
+    @Description("Verifies that artifacts can only be added as long as the artifact storage quota is not exceeded.")
+    public void uploadArtifactsUntilStorageQuotaExceeded() throws Exception {
+        
+        final long storageLimit = quotaManagement.getMaxArtifactStorage();
+
+        final int artifactSize = 64 * 1024;
```

Comment:
The same reasoning as above for ArtifactManagementTest.java

Reply:
done